content/reference/admin/private_locations/configuration/index.md

```diff
@@ -65,14 +65,22 @@ The status has three possible values:
 The control plane agent is distributed as a docker image: [`gatlingcorp/control-plane`](https://hub.docker.com/r/gatlingcorp/control-plane).
 It is configured by a file mounted at `/app/conf/control-plane.conf`.
 
+For example of deployments, see:
+* [Amazon elastic container service]({{< ref "../ecs_deployment" >}})
+* [Kubernetes deployment]({{< ref "../kubernetes_deployment" >}})
+
+### Control plane configuration
+
+Only control plane and locations ID, description and type are transferred to Gatling Enterprise Cloud.
+
 {{< alert info >}}
 The configuration file uses the [HOCON format (Human-Optimized Config Object Notation)](https://github.com/lightbend/config/blob/master/HOCON.md):
 {{< /alert >}}
 
-Example with an AWS location (only supported type at the moment):
-{{< include-code "control-plane.conf" bash >}}
-
-Only control plane and locations ID, description and type are transferred to Gatling Enterprise Cloud.
+{{< include-file >}}
+AWS: includes/control-plane.aws.md
```

Comment:
```suggestion
AWS EC2: includes/control-plane.aws.md
```
Might be worth being more explicit, since the Kubernetes option can also be used on AWS (with e.g. EKS)?